src/Deferred.php

```diff
@@ -41,7 +42,7 @@ final class Deferred implements Interfaces\Deferred
     private $output;
 
     /**
-     * @var \Throwable
+     * @var ?\Throwable
```

Comment:
In docblock tags it is `\Throwable|null`

Reply:
Yes it does but if you look a docblocks in other projects, question mark is not used in for `null`.
https://docs.phpdoc.org/guide/references/phpdoc/tags/return.html#return

Reply:
https://github.com/phpstan/phpstan-src/blob/80ce9a404787133ff3fa266f90602728bf1c349c/src/Node/Printer/ExprPrinter.php#L17

Reply:
Do haven't written `?int` below.

Reply:
Well, I wasn't going to let it be a docblock forever :slightly_smiling_face: 

#16